src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -95,34 +93,59 @@ public void setup() throws InterruptedException, ServiceBusException, JSONExcept
             .atMost(60, TimeUnit.SECONDS)
             .pollDelay(2, TimeUnit.SECONDS)
             .until(() -> isExceptionRecordCreated(randomPoBox));
-
     }
 
     @Test
-    @Disabled
     public void should_atatch_exception_record_to_the_existing_case() {
+        //given
+        String caseData = SampleData.fileContentAsString("envelopes/new-envelope.json");
+        Envelope newEnvelope = EnvelopeParser.parse(caseData);
+        caseDetails = ccdCaseCreator.createCase(newEnvelope);
+
+        // when
+        callCallbackUrlToAttachExceptionToCase(caseDetails, exceptionRecord);
+
+        //then
+        await("Exception record is attached to the case")
+            .atMost(60, TimeUnit.SECONDS)
+            .pollDelay(2, TimeUnit.SECONDS)
+            .until(() -> isExceptionRecordAttachedToTheCase(1));
+    }
+
+    @Test
+    public void should_atatch_exception_record_to_the_existing_case_with_evidence_documents() throws JSONException {
+        //given
+        Envelope newEnvelope = updateEnvelope("envelopes/new-envelope-with-evidence.json");
+        caseDetails = ccdCaseCreator.createCase(newEnvelope);
 
         // when
+        callCallbackUrlToAttachExceptionToCase(caseDetails, exceptionRecord);
+
+        //then
+        await("Exception record is attached to the case")
+            .atMost(60, TimeUnit.SECONDS)
+            .pollDelay(2, TimeUnit.SECONDS)
+            .until(() -> isExceptionRecordAttachedToTheCase(2));
+    }
+
+    private void callCallbackUrlToAttachExceptionToCase(CaseDetails caseDetails, CaseDetails exceptionRecord) {
```

Comment:
may be better call it as invokeCallbackEndpointForLinkingDocsToCase ?

Reply:
renamed